system-probe_arm64/Dockerfile

```diff
@@ -37,4 +42,10 @@ RUN mkdir -p $GOPATH/src/github.com/DataDog/datadog-agent
 RUN wget "https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.0/clang+llvm-11.0.0-aarch64-linux-gnu.tar.xz" -O /tmp/clang.tar.xz  -o /dev/null
 RUN mkdir -p /opt/clang
 RUN tar xf /tmp/clang.tar.xz  -C /opt/clang --strip-components=1
-ENV PATH /opt/clang/bin:${PATH}
```

Comment:
Why did this need to be moved to the entrypoint script?

Reply:
I copy-pasted the original `entrypoint.sh` script, which has the following shebang: `#!/bin/bash -l`. The `-l` option (login shell) resets the `PATH` at the start of the script, which is why I had to do the move.

I'll check if the `-l` option is really needed. If it's not, I'll remove it and add back the `ENV` statement.